manifests/server/options.pp

```diff
@@ -20,6 +20,7 @@
 define dns::server::options(
   $forwarders = [],
   $listen_on = [],
+  $listen_on_port = undef,
```

Comment:
I've created a separate pull request to fix up the other documentation for this class.  If you want to merge that one first, I'll rebase this and make the appropriate changes to match the docs in the other pull request.